README.md

```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
 ninja install-c2s
 ```
 
-### Deployment
+Note: For computers with 16G and below memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
+```
+ninja -j4 install-c2s
+```
+
+
+**build success message**:
+
+After the compilation is successful, the following information will be outputted in terminal.
```

Comment:
After a successful build, you should be able to see following message in the terminal output.